backend/src/test/java/eventlistener/controller/NotificationUserControllerTest.java

```diff
@@ -161,16 +165,43 @@ void addUserGetNoEvent() {
     @Test
     void getSingleUser() {
         //GIVEN
-        NotificationUser userToFind = NotificationUser.builder()
-                .id("1")
+        NotificationUser userToAdd = NotificationUser.builder()
                 .name("TestUser")
                 .email("test@test.de")
                 .build();
-        notificationUserRepo.save(userToFind);
+        NotificationUser expected = notificationUserRepo.save(userToAdd);
+        //WHEN
+        ResponseEntity<NotificationUser> responseEntity = restTemplate.exchange("/api/user/"+expected.getId(), HttpMethod.GET , new HttpEntity<>(getLoginHeader()), NotificationUser.class);
+        //THEN
+        assertThat(responseEntity.getStatusCode(), is(HttpStatus.OK));
+        assertThat(responseEntity.getBody(), is(expected));
+    }
+
+    @Test
+    @DisplayName("Edit an existing User")
+    void testEditUser() {
+        //GIVEN
+        NotificationUser userToEdit = NotificationUser.builder()
+                .name("TestUser")
+                .email("test@test.de")
+                .build();
+        NotificationUser userWithId = notificationUserRepo.save(userToEdit);
+        Event userEvent = eventRepo.save(Event.builder().name("TestEvent").description("Beschreibung").actions(List.of(Action.MAIL)).notificationUser(List.of(userWithId)).build());
+        NotificationUserEditDTO newUser = NotificationUserEditDTO.builder()
```

Comment:
I think the tests would be much more readable if you put all your builder()...build() in a method in the testhelper as well